Gemfile

```diff
@@ -5,13 +5,18 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
-# Use Puma as the app server
-gem 'puma', '~> 5.0'
+gem 'haml',                 '~> 4.0'
+gem 'puma',                 '~> 5.0'
+gem 'rake',                 '~> 13.1'
+gem 'sinatra',              '~> 3.0'
+gem 'sinatra-param',        '~> 1.6'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
-  gem 'rspec', '~> 3.10'
+  gem 'rack-test', '~> 2.1'
+  gem 'rspec',     '~> 3.10'
```

Comment:
Rubocop has an add-on for rspec that helps us write cleaner and nicer tests
https://github.com/rubocop/rubocop-rspec

Reply:
Nice. I will check it out!